sql/updates/post-update.sql

```diff
@@ -171,3 +171,40 @@ $$;
 -- Repair relations that have relacl entries for users that do not
 -- exist in pg_authid
 CALL _timescaledb_functions.repair_relation_acls();
+
+-- Migrate existing CAggs using time_bucket_ng to time_bucket
+CREATE OR REPLACE PROCEDURE _timescaledb_functions.cagg_migrate_to_time_bucket(cagg REGCLASS)
+   AS '@MODULE_PATHNAME@', 'ts_continuous_agg_migrate_to_time_bucket' LANGUAGE C;
+
+DO $$
+DECLARE
+  cagg regclass;
+  ts_major INTEGER;
+  ts_minor INTEGER;
+BEGIN
+  SELECT
+    ((string_to_array(extversion,'.'))[1])::int,
+    ((string_to_array(extversion,'.'))[2])::int
+  INTO
+    ts_major, ts_minor
+  FROM
+    pg_catalog.pg_extension
+  WHERE
+    extname = 'timescaledb';
+
+  -- CAggs supporting time_bucket with orign and/or offset was released on 2.15.X
+  IF ts_major >= 2 AND ts_minor >= 15 THEN
```

Comment:
I don't remember exactly how the update scripts work, but post-update sounds as if it is _after_ the update, will the version check really be correct then? Or, are we still on the old version here?

Reply:
I think we don't need this version check because this change will be part either of next 2.18.3 or 2.19.0 that both already have all necessary functions/metadata to properly execute the `cagg_migrate_to_time_bucket` function.